packages/geolocation/.editorconfig

```diff
@@ -0,0 +1,16 @@
+# Editor configuration, see https://editorconfig.org
+root = true
+
+[*]
+charset = utf-8
+indent_style = space
+indent_size = 2
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.js]
+quote_type = single
+
+[*.md]
+max_line_length = 75
+trim_trailing_whitespace = false
```

Comment:
@Edilberto-Vazquez can we add the EOL please?

Reply:
I already add the EOL